contracts/neutron_interchain_txs/src/storage.rs

```diff
@@ -42,6 +44,17 @@ pub fn read_reply_payload(store: &mut dyn Storage) -> StdResult<SudoPayload> {
     from_binary(&Binary(data))
 }
 
+pub fn add_error_to_queue(store: &mut dyn Storage, error_msg: String) -> StdResult<()> {
+    read_errors_from_queue(store)
```

Comment:
micro optimization, instead of collecting all the errors you get the last key with `ERRORS_QUEUE.keys(store, None, None, Order::Descending).next();`

Reply:
Implemented